apps/prairielearn/src/ee/lib/lti13.ts

```diff
@@ -838,10 +838,14 @@ export async function updateLti13Scores(
   const assessment_instances = await queryRows(
     sql.select_assessment_instances_for_scores,
     { assessment_id: assessment.id },
-    AssessmentInstanceSchema.extend({
-      score_perc: z.number(), // not .nullable() from SQL query
-      date: DateFromISOString, // not .nullable() from SQL query
-      users: UserWithLti13SubSchema.array(),
+    z.object({
+      id: IdSchema,
+      user_id: IdSchema,
+      assessment_id: IdSchema,
```

Comment:
AFAICT, both of these are unused. Can we remove them from the query and this type?

Reply:
In b37f006 I changed the query to only return the columns that are needed instead of `cai.*`.